packages/page-staking/src/Nomination/useBalance.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState, useEffect } from 'react';
+import {useApi, useCall} from '@polkadot/react-hooks';
+import { DeriveBalancesAll } from '@polkadot/api-derive/types';
+import { formatNumber } from '@polkadot/util';
+import { Balance } from '@polkadot/types/interfaces/runtime';
+import BN from 'bn.js';
+import { formatBalance } from '@polkadot/util';
+
+// Known account we want to use (available on dev chain, with funds)
+
+export function useBalance (address?: string | null): string | null  {
+  const api = useApi();
+  const balancesAll = useCall<DeriveBalancesAll>(api.api.derive.balances.all as any, [address]);
+  return balancesAll ? formatNumber(balancesAll.freeBalance) : null
+}
+
+export function useBalanceClear (address?: string | null): Balance | null  {
+  const api = useApi();
+  const balancesAll = useCall<DeriveBalancesAll>(api.api.derive.balances.all as any, [address]);
+  return balancesAll ? balancesAll.availableBalance : null
+}
+
+export type WholeFeesType = {
+  wholeFees: Balance;
+  feesLoading: boolean;
+}
```

Comment:
Move to the top after the impports, so it is explicit which types are in the file.